### ISSUE TITLE
NullReference in PlatformSettings (having only FluentAvalonia in the Application.Styles)

### ISSUE THREAD

**Issue by PhilWeisz** [amwx/FluentAvalonia#452]
**Describe the bug**
There is a NullReference-exception showing that the PlatformSettings are null in the method 'ResolveThemeAndInitializeSystemResources()'

**Screenshots**
Error + callstack:
![image](https://github.com/amwx/FluentAvalonia/assets/122535935/7f2a0d9b-c5be-42f7-9ad8-2a3ecb921d01)
![image](https://github.com/amwx/FluentAvalonia/assets/122535935/55a6d216-b9e4-4af4-9197-933aa89cbcac)

Application Styles:
![image](https://github.com/amwx/FluentAvalonia/assets/122535935/61949115-3779-4694-aa6c-ce3fecb3e678)

**Desktop/Platform (please complete the following information):**
 - OS: Windows (11/10)
 - FluentAvalonia Version 2.0.0
 - Avalonia Version 11.0.2

**Additional context**
Am using Microsoft DI Extensions, Boostrapper for configuring DI

**Comment by amwx** [comment]
Try with the new 2.0.1 package. I really don't know why this is occurring though. The docs say PlatformSettings is non-null after the application instance initializes which occurs before xaml is parsed and that method is called, so the fact that it is null means something is wrong. I haven't had any issues and no other issues have come up (that I'm aware of), so my suggestion would be to maybe start with a new project and start adding your components slowly back to see if it breaks, then that might indicate why this is happening

**Comment by PhilWeisz** [comment]
In the new project it indeed doesn't happen, we found out it happens when you manually create the App.Ctor and call the override of Initialize() in it (which in turn just does the AvaloniaXamlLoader.Load(this)). Then it's not the PlatformSettings that are null - but the Application.Current <- returned null (which actually is GetService<Application>())  - We now think it doesn't have anything to do with FluentAvalonia - but it crashed in FA since it does try to access the Application.Current-instance. Thank you for your time and sorry! Closing.
